File: tests/wiringPi/softTone.py
import importlib
import pytest

m=importlib.import_module("wiringop.wiring_pi.soft_tone")

def test_soft_tone_create():
    _=m.soft_tone_create(1)

def test_soft_tone_stop():
    _=m.soft_tone_stop(1)

def test_soft_tone_write():
    _=m.soft_tone_write(1, 1)

File: support/mock_backend/wiringPi/softTone.c
#include <stdint.h>
#include <stddef.h>
#include <stdbool.h>
#include <string.h>
#include "wiringPi/softTone.h"
int softToneCreate(int pin) { (void)pin;return 1; }
void softToneStop(int pin) { (void)pin; }
void softToneWrite(int pin, int freq) { (void)pin;(void)freq; }
